packages/puppeteer-core/src/common/TouchError.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class TouchError extends Error {}
```

Comment:
This should live in `Errors.ts` and extend PuppeteerError for user facing errors.

Reply:
done